PWGCF/TwoParticleCorrelations/Tasks/longrangecorrDerived.cxx

```diff
@@ -103,19 +103,33 @@ struct LongrangecorrDerived {
   OutputObj<CorrelationContainer> same{"sameEvent"};
   OutputObj<CorrelationContainer> mixed{"mixedEvent"};
 
+  using CollTable = aod::CollLRTables;
+  using TrkTable = aod::TrkLRTables;
+  using MftTrkTable = aod::MftTrkLRTables;
+  using Ft0aTrkTable = aod::Ft0aLRTables;
+  using Ft0cTrkTable = aod::Ft0cLRTables;
+  using MftbestTrkTable = aod::MftBestTrkLRTables;
+  using V0TrkTable = aod::V0TrkLRTables;
```

Comment:
I would keep the plurals because they are not iterators
But not in the same way as in `LR`
Is `Colls` what should be plural and not `Tables`
The same for the others